package.json

```diff
@@ -4,7 +4,11 @@
   "description": "",
   "main": ".eslintrc.js",
   "dependencies": {
-    "eslint-config-hackreactor": "^1.0.0"
+    "axios": "^0.19.2",
+    "eslint-config-hackreactor": "^1.0.0",
+    "file-system": "^2.2.2",
+    "mongoose": "^5.9.4",
+    "path": "^0.12.7"
```

Comment:
It seems like some of these dependencies are not needed (yet). I'm just reiterating what Josh said, which was to install packages and such as you need them.
Better organization of package versions. Less bulk on the application, etc.
Path and file-system are built in the node aren't they? Is there a reason to grab additional packages for them?

Reply:
I didn't know file-system and path were native to node, I'll look into removing those. I was using axios to grab my thousand pictures so its just lingering from that. I'll do my best to keep my dependencies low.